framework/Samplers/Importance.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  This module contains the Umbrella sampling strategy
+
+  Created on Feb 15, 2021
+  @author: Tanaya
+  supercedes Samplers.py from crisr
+"""
+#for future compatibility with Python 3--------------------------------------------------------------
+from __future__ import division, print_function, unicode_literals, absolute_import
+#End compatibility block for Python 3----------------------------------------------------------------
+
+#External Modules------------------------------------------------------------------------------------
+import numpy as np
+from operator import mul
+from functools import reduce
+#External Modules End--------------------------------------------------------------------------------
+
+#Internal Modules------------------------------------------------------------------------------------
+from .ForwardSampler import ForwardSampler
+from utils import utils,randomUtils,InputData, InputTypes
+#Internal Modules End--------------------------------------------------------------------------------
+
+class Importance(ForwardSampler):
```

Comment:
Is there a regression test added for this Sampler?

Reply:
Regression test has been added